doc/io_chaos.md

```diff
@@ -0,0 +1,157 @@
+
+#### Data
+
+The data directory of the component should be a subdirectory of `PersistentVolumes`.
+
+### Run
+
+Assuming user are using `examples/io-mixed-sample.yaml`, to create a chaos experiment:
+
+```bash
+kubectl apply -f examples/io-mixed-sample.yaml
+```
+
+## Spec Arguements
+
+* **selector**: is used to select pods that are used to inject chaos action.
+
+* **action**: action represents the chaos action about IO action, now the **delay**, **errno**,  **mixed** action is supported. User can go to [*IO Chaos Availiable Actions*](#io-chaos-availiable-actions) for more details.
+* **mode**: Mode defines the mode to run chaos action. Supported mode: `one` / `all` / `fixed` / `fixed-percent` / `random-max-percent`.
+* **duration**: represents the duration of the chaos action. The duration is a possibly string with signed sequence of decimal numbers,  each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `”2h45m"`.
+* **delay**: defines the value of I/O chaos action delay. The duration is a possibly string with signed sequence of decimal numbers,  each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `”2h45m”`. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+  If `Delay` is empty, the operator will generate a value for it randomly.
+* **errno**: defines the error code that returned by IO action. It is a int32 string like `"32"`. This field should be set when user choose `errno`  or `mixed` action. If `errno` is empty, the operator will generate a error code for it randomly. User can set the `errno` refer to: https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/Errors/unix_system_errors.html.
+* **percent**: Percent defines the percentage of injection errors and provides a number from 0-100. The defualt value is `100`.
+* **path**: defines the path of files for injecting I/O chaos action. It should be an regular expression for the path user want to inject errno or delay. If path is `""` or not defined, IO to all files will be injected.
+* **methods**: defines the I/O methods for injecting I/O chaos action. It’s an array of string, which set the IO syscall like `open` `read`. User can see the [availiable methods](#availiable-methods) below.
+* **addr**: defines the sidecar HTTP server address for sidecar container, like `":8080"`.
+* **configName**: defines the config name which used to inject pod. User can refer to [examples/tikv-configmap.yaml](../../examples/tikv-configmap.yaml) to define user's config.
+* **layer**: represents the layer of the I/O action. Supported value: `fs` , and default is `fs`.
+
+## IO Chaos Availiable Actions
+
+IO Chaos now support the actions below:
+
+* **delay**: IO delay action. User can specify the latency before the IO operation will return.
+* **errno**: IO errno action. In this mode read/write IO operation will return error.IO errno means user's read/write IO operations will return error.
```

Comment:
```suggestion
* **errno**: IO errno action. In this mode, read/write IO operations will return an error.
```
I deleted the sentence because it seems to duplicate the previous sentence. How do you think?